bigquery/cloud-client/README.rst.in

```diff
@@ -16,31 +16,10 @@ setup:
 - install_deps
 
 samples:
-- name: Simple Application
-  file: simple_app.py
 - name: Quickstart
   file: quickstart.py
-- name: Query
-  file: query.py
-  show_help: true
-- name: Parameterized Query
-  file: query_params.py
-  show_help: true
-- name: Snippets
-  file: snippets.py
-  show_help: true
-- name: Load data from a file
-  file: load_data_from_file.py
-  show_help: true
-- name: Load data from Cloud Storage
-  file: load_data_from_gcs.py
-  show_help: true
-- name: Load streaming data
-  file: stream_data.py
-  show_help: true
-- name: Export data to Cloud Storage
-  file: export_data_to_gcs.py
-  show_help: true
+- name: Simple Application
+  file: simple_app.py
```

Comment:
Could you add `user_credentials.py` too? I guess I forgot to add that here when I added that sample.

Reply:
Done